generators/excel-schema/main.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+import os
+import csv
+import openpyxl
+import sys
+
+OPENPYXL_MAJOR_VERSION = int(openpyxl.__version__[0])
+DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S%Z'
+# Minimum possible header.
+HEADER = (
+    'domain',
+    'entity',
+    'attribute',
+    'value',
+)
+
+
+def generate(file_name, domain):
+    # Creating a CSV writer that feeds to stdout.
+    w = csv.writer(sys.stdout)
+
+    # Write the header to start.
+    w.writerow(HEADER)
+
+    wb = openpyxl.load_workbook(file_name, use_iterators=True)
+    sheets = wb.get_sheet_names()
+
+    for i, sheet_name in enumerate(sheets):
+        sheet = wb.get_sheet_by_name(sheet_name)
+        column_names = _column_names(wb, sheet_name)
+        rows = sheet.iter_rows()
+
+        # Skip header row because we already collected the column names.
+        header = next(rows)
+
+        for j, cell in enumerate(header):
+            label = list(column_names)[j]
+            column_alpha = cell.column
```

Comment:
it's the alphabetical label for the cell. 'A', 'B', CDD'. I can name it something more self explanatory